docs/pages/versions/unversioned/sdk/audio.md

```diff
@@ -513,11 +513,31 @@ A static convenience method to construct and start a recording is also provided:
 
   A `Promise` that is fulfilled when the recorder is loaded and prepared, or rejects if this failed. If another `Recording` exists in your experience that is currently prepared to record, the `Promise` will reject. If the `RecordingOptions` provided are invalid, the `Promise` will also reject. The promise is resolved with the `status` of the recording (see `getStatusAsync()` for details).
 
-- `recordingInstance.isPreparedToRecord()`
```

Comment:
This method doesn't seem to exist anymore. There is a note about it in the test suite: https://github.com/expo/expo/blob/27822831ad462b5d8cf1398ee74c60f7248bfe42/apps/test-suite/tests/Recording.js#L128